api/src/test/java/org/apache/iceberg/io/TestCloseableGroup.java

```diff
@@ -91,7 +92,7 @@ public void notSuppressExceptionIfSetSuppressIsFalse() throws Exception {
     closeableGroup.addCloseable(closeable2);
     closeableGroup.addCloseable(closeable3);
 
-    Assertions.assertThatThrownBy(closeableGroup::close).isEqualTo(ioException);
+    assertThatThrownBy(closeableGroup::close).isEqualTo(ioException);
```

Comment:
do we really need these changes? It just adds unnecessary overhead when reviewing

Reply:
Yes we don't actually need these changes. Made it so just as to standardize the imports across the tests. But sure...will revert these changes.